.github/renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    ":disableDependencyDashboard",
+    ":semanticCommitsDisabled"
+  ],
+  "prHourlyLimit": 10,
```

Comment:
TODO: This will be removed before merging. I've added it so that renovate creates everything quick to be able to test the "final picture".